kyo-core/shared/src/main/scala/kyo/scheduler/IOTask.scala

```diff
@@ -34,6 +34,8 @@ sealed private[kyo] class IOTask[Ctx, E, A] private (
 
     private inline def erasedAbortTag = Tag[Abort[Any]].asInstanceOf[Tag[Abort[E]]]
 
+    private inline def locally[A](inline f: A): A = f
```

Comment:
`locally` is used here just as a workaround for a scalafmt bug that makes the last line of handling functions be moved to the wrong indentation. It was also allocating a funciton.